packages/lib/event-types/getEventTypesByViewer.ts

```diff
@@ -31,7 +31,7 @@ type User = {
 type Filters = {
   teamIds?: number[];
   upIds?: string[];
-  schedulingTypes?: SchedulingType[];
+  schedulingTypes?: (SchedulingType | null)[];
```

Comment:
> Yes would be the same but I did it because I couldn't shake the type error without early return.
> 
> ![Screenshot 2024-05-16 at 1 49 39 PM](https://private-user-images.githubusercontent.com/74371312/331113506-71694bfb-f78a-4729-899a-2180a81af3ef.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU4NTI1MjQsIm5iZiI6MTcxNTg1MjIyNCwicGF0aCI6Ii83NDM3MTMxMi8zMzExMTM1MDYtNzE2OTRiZmItZjc4YS00NzI5LTg5OWEtMjE4MGE4MWFmM2VmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTE2VDA5MzcwNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWRmNWU3MDM0OTkwNmU1NjFmMGM5ZDUyMzA2NjlkMTMwNzA2ZmM4ODIyZjIwMTUxMTlhM2U0ZjgzOWQ0ZGRjMDgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.qeM16jgWKHTP4bETl5eGXWNP4ftgktJKKByT7zhlbCo)


Then we get this type error